src/select/Select.ts

```diff
@@ -231,12 +231,21 @@ export default class Select extends SelectBase<SelectProperties> {
 	@onPropertiesChanged()
 	protected onPropertiesChanged(evt: PropertiesChangeEvent<this, SelectProperties>) {
 		const {
-			customOption = SelectOption
+			customOption = SelectOption,
+			options = []
 		} = this.properties;
 
+		// update custom option registry
 		if ( !this.registry || includes(evt.changedPropertyKeys, 'customOption')) {
 			this.registry = this._createRegistry(customOption);
 		}
+
+		// add ids to options for use with aria-activedescendant
+		if (includes(evt.changedPropertyKeys, 'options')) {
+			for (let option of options) {
```

Comment:
should we be using native for...of or the [forOf shim](https://github.com/dojo/shim/blob/master/src/native/iterator.ts#L66)